src/hardware/serialport/softmodem.cpp

```diff
@@ -187,11 +259,13 @@ void CSerialModem::SendRes(const ResTypes response) {
 	}
 }
 
-bool CSerialModem::Dial(char * host) {
+bool CSerialModem::Dial(const char * host) {
+	char buf[128];
+	strcpy(buf, host);
```

Comment:
```
safe_strcpy(buf, host);
```
Our `safe_strcpy` wrapper will take care of determining size of destination buffer in this case.